privacyidea/lib/importotp.py

```diff
@@ -500,7 +499,9 @@ def parsePSKCdata(xml_data,
                 enc_data = enc_data.strip()
                 secret = aes_decrypt_b64(binascii.unhexlify(preshared_key_hex), enc_data)
                 if token["type"].lower() in ["hotp", "totp"]:
-                    token["otpkey"] = binascii.hexlify(secret)
+                    token["otpkey"] = hexlify_and_unicode(secret)
+                elif token["type"].lower() in ["pw"]:
+                    token["otpkey"] = to_unicode(secret)
                 else:
                     token["otpkey"] = secret
```

Comment:
Hm, I think ``secret`` is a bytestring in this case. I'm not sure if this is supported by the underlying ``import_token`` API?

... but on a second thought, can we even import any token types besides HOTP, TOTP and PW?

Reply:
We can import Spass tokens. Which have no otpkey anyways.

Reply:
done in d367ea32